include/freerdp/settings.h

```diff
@@ -1691,7 +1692,8 @@ extern "C"
 		ALIGN64 char* ActionScript;           /* 5195 */
 		ALIGN64 UINT32 Floatbar;              /* 5196 */
 		ALIGN64 UINT32 TcpConnectTimeout;     /* 5197 */
-		UINT64 padding5312[5312 - 5198];      /* 5198 */
+		ALIGN64 UINT32 FakeMouseMotionInterval; /* 5198 */
```

Comment:
* you need to run `./tools/update-settings-tests` and `./tools/update-rdpSettings` if you change the `rdpSettings` struct.
* changes need to be formatted with `git clang-format` before committing them.